### ISSUE TITLE
[Accessibility][Bug] First row of table item is skipped when using tab navigation in table with fdpTableColumnResponsive

### ISSUE THREAD

**Issue by karolgromacki** [SAP/fundamental-ngx#13603]
### Describe your proposal

The first part of a row is not focusable using the Tab key. Instead, it requires arrow key navigation, while the second part of the row is accessible via Tab. This creates an inconsistent and confusing keyboard navigation experience.

Expected Behavior: 
All interactive elements within a table row should be navigable consistently using the Tab key, even at 200% text size, ensuring full keyboard accessibility.

<img width="3452" height="1734" alt="Image" src="https://github.com/user-attachments/assets/d175bb45-f99b-45df-990a-957623533d5f" />

https://github.com/user-attachments/assets/102c169f-8009-4596-b358-d6bdb0bf27d0

### Which versions of Angular and Fundamental Library for Angular are affected?

Angular: 20.1.4, Fundamental Library: 0.58.0-rc.16

### Steps to Reproduce (for bugs)

1. Go to https://fundamental-ngx.netlify.app/#/platform/table/basic
2. Scroll to "Hiding columns depending on the table size" section.
3. Select table width: 800 pixels.
4. Navigate to the first table item using tab navigation.
5. Observe that the item is divided to 2 rows and the second is selected instead of the first.

### Declaration

- [x] I'm not disclosing any internal or sensitive information.

**Comment by mikerodonnell89** [comment]
duplicate of #13174 fixed with #14119
